entity/contents/filter_contents.h

```diff
@@ -91,4 +106,37 @@ class BlendFilterContents : public FilterContents {
   FML_DISALLOW_COPY_AND_ASSIGN(BlendFilterContents);
 };
 
+/*******************************************************************************
+ ******* GaussianBlur1DFilterContents
+ ******************************************************************************/
+
+class GaussianBlur1DFilterContents final : public FilterContents {
+ public:
+  GaussianBlur1DFilterContents();
+
+  ~GaussianBlur1DFilterContents() override;
+
+  void SetRadius(Scalar radius);
+
+  void SetDirection(Point direction);
+
+  void SetExpandBorder(bool expand);
```

Comment:
I really can't tell what this is meant to do from looking at the filter documentation. This doesn't seem to be a general concept.

Reply:
Checking out Skia, it looks like the blur filter always "expands" the image bounds by the radius. I went ahead and inverted this behavior and switched to "clip" as the terminology instead. I can back it out instead for now if you'd prefer.